target/linux/mpc85xx/p1020/config-default

```diff
@@ -54,3 +53,5 @@ CONFIG_UBIFS_FS_ZLIB=y
 CONFIG_UBIFS_FS_ZSTD=y
 CONFIG_WS_AP3710I=y
 CONFIG_XPS=y
+CONFIG_HIVEAP_330=y
+CONFIG_HIVEAP_370=y
```

Comment:
Sorting.

Reply:
A question here actually, I don't really want to define CONFIG_HIVEAP_330 when selecting the device configuration, but if I don't define both 330 and 370 here, it asks every build if I want to include either 330 or 370

Reply:
HIVEAP_330 is already defined in that file, you just move it down.